src/App.jsx

```diff
@@ -1,13 +1,37 @@
-import { useState } from 'react'
+import { useState, useEffect } from 'react'
+import { Routes, Route } from 'react-router-dom';
+import Loading from './Page-Components/Loading'
 import './css/App.css'
+import Header from './Components/Header'
+import Home from './Page-Components/Home'
+import fetchApiData from './api';
+import AllArticles from './Page-Components/AllArticles';
+
 
 function App() {
+  const [articles, setArticles] = useState([])
+  const[isLoading, setIsLoading] = useState(true)
+
+  useEffect(() => {
+     fetchApiData.get('/articles')
```

Comment:
You could put your API functions in the api.js file, i.e. make a "getArticles" function in your API folder which makes the .get request and extracts the data, then invoke that function here.
This would make your components a bit neater but isn't mandatory